Content.Shared/_ES/Masks/Aura/ESSenseAuraSystem.cs

```diff
@@ -0,0 +1,52 @@
+using System.Linq;
+using Content.Shared._ES.Objectives;
+using Content.Shared.Actions;
+using Content.Shared.Mind;
+using Content.Shared.Popups;
+using Content.Shared.Whitelist;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared._ES.Masks.Aura;
+
+public sealed class ESSenseAuraSystem : EntitySystem
+{
+    [Dependency] private readonly IPrototypeManager _prototype = default!;
+    [Dependency] private readonly EntityWhitelistSystem _entityWhitelist = default!;
+    [Dependency] private readonly SharedMindSystem _mind = default!;
+    [Dependency] private readonly ESSharedObjectiveSystem _objective = default!;
+    [Dependency] private readonly SharedPopupSystem _popup = default!;
+
+    /// <inheritdoc/>
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<ESSenseAuraEvent>(OnSenseAuraEvent);
+    }
+
+    private void OnSenseAuraEvent(ESSenseAuraEvent args)
+    {
+        // This can't be predicted but there's no harm in having it in shared.
+        if (!_mind.TryGetMind(args.Target, out var mind, out _))
```

Comment:
This whole thing feels.. overly abstract to me? I get the intent but there should probably be an override for it on individual masks (not on the aura prototype), which would also solve the "no objectives" problem.

Additionally, if being generalized this way, the senser should probably have a list of auras they recognize so you can have auras for use only by other aura-based masks (empath probably doesnt need to recognize a "fruity" aura, while the Fruit Destroyer probably does.)